pkg/port/cli/integration.go

```diff
@@ -38,6 +38,20 @@ func (c *PortClient) CreateIntegration(i *port.Integration) (*port.Integration,
 	return &pb.Integration, nil
 }
 
+func (c *PortClient) GetIntegration(stateKey string) (*port.Integration, error) {
+	pb := &port.ResponseBody{}
+	resp, err := c.Client.R().
+		SetResult(&pb).
+		Get(fmt.Sprintf("v1/integration/%s", stateKey))
+	if err != nil {
+		return nil, err
+	}
+	if !pb.OK {
+		return nil, fmt.Errorf("failed to get integration, got: %s", resp.Body())
+	}
+	return &pb.Integration, nil
+}
+
 func (c *PortClient) GetIntegrationConfig(stateKey string) (*port.AppConfig, error) {
 	pb := &port.ResponseBody{}
```

Comment:
Why not reusing `GetIntegration`?

Reply:
Ill remove it on the other pr
it will be easier